spanner/spannertest/inmem.go

```diff
@@ -640,6 +674,25 @@ func (s *server) readStream(ctx context.Context, tx *transaction, send func(*spa
 	return nil
 }
 
+func (s *server) buildResultSetMetaData(ri rowIter) (*spannerpb.ResultSetMetadata, error) {
```

Comment:
"Metadata"

everything else here treats that as a single word.

Reply:
Done.